en/components/grids_templates/sorting.md

```diff
@@ -25,8 +25,6 @@ _keywords: angular sort, ignite ui for angular, infragistics
 In Ignite UI for Angular @@igComponent, data sorting is enabled on a per-column level, meaning that the **@@igSelector** can have a mix of sortable and non-sortable columns. Performing angular sort actions enables you to change the display order of the records based on specified criteria.
 
 #### Demo
-Additionally there is a custom contextmenu added for sorting using **@@igSelector**'s [`onContextMenu`]({environment:angularApiUrl}/classes/@@igTypeDoc.html#oncontextmenu) Output.
```

Comment:
This line should be removed only for IgxGrid, because the other topics do not have a section for [context menu](https://staging.infragistics.com/products/ignite-ui-angular/angular/components/grid/selection.html#grid-context-menu) and this is their only context menu example.